### PR TITLE
Revert "Added missing recipes for weaponsmithing from Chapter 7."

### PR DESCRIPTION
**Pull request by jayRyan24**
Reverts rpherbig/dr-scripts#4656

This needs to be reverted and then rechecked for ones that are missing. It's breaking Smith by asking people to choose from multiple entries of the same item.

### CODE REVIEW

**Comment by jayRyan24** [comment]
@Dartellum

**Comment by Dartellum** [comment]
I can look through the file again.

File: data/base-recipes.yaml [hunk 1]
```diff
@@ -4053,142 +4053,6 @@ crafting_recipes:
   chapter: 7
   part:
   - long pole
-- name: a metal glaive
```
Comment (Dartellum) [inline data/base-recipes.yaml:4056]:
```yaml
- name: a metal javelin
  noun: javelin
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a light metal spear
  noun: spear
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal scythe
  noun: scythe
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal bardiche
  noun: bardiche
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal lochaber axe
  noun: axe
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal guisarme
  noun: guisarme
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal halberd
  noun: halberd
  volume: 12
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal tzece
  noun: tzece
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal coresca
  noun: coresca
  volume: 12
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal awgravet ava
  noun: ava
  volume: 7
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal hunthsleg
  noun: hunthsleg
  volume: 10
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal allarh
  noun: allarh
  volume: 10
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal spear
  noun: spear
  volume: 10
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal glaive
  noun: glaive
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a two-pronged halberd
  noun: halberd
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal khuj
  noun: khuj
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal partisan
  noun: partisan
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal military fork
  noun: fork
  volume: 10
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal duraka skefne
  noun: skefne
  volume: 10
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal pole axe
  noun: axe
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal fauchard
  noun: fauchard
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal ngalio
  noun: ngalio
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal pike
  noun: pike
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal lance
  noun: lance
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal spetum
  noun: spetum
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal ranseur
  noun: ranseur
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal ilglaiks skefne
  noun: skefne
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal voulge
  noun: voulge
  volume: 13
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a metal awl pike
  noun: pike
  volume: 15
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
- name: a short-hafted halberd
  noun: halberd
  volume: 9
  type: weaponsmithing
  work_order: true
  chapter: 7
  part:
  - long pole
  ```
  Here they are in page order and no duplicates. My mistake was thinking they were in page order and I started at the last one entered.

Comment (Dartellum) [reply data/base-recipes.yaml:4056]: Instead of reverting, just merge in the updated list with this PR.

**Comment by Dartellum** [comment]
Should I put in a correction PR for this or will you add what I put in above?

**Comment by rpherbig** [comment]
@Dartellum @jayRyan24 let me know what you two come up with

**Pull request closed without merging**